sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/AuthenticationsImpl.java

```diff
@@ -89,6 +89,16 @@
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<AcrRefreshToken>> exchangeAadAccessTokenForAcrRefreshTokenWithResponseAsync(
             String serviceParam, String accessToken) {
+        if (this.client.getUrl() == null) {
+            return Mono.error(
+                    new IllegalArgumentException("Parameter this.client.getUrl() is required and cannot be null."));
+        }
+        if (serviceParam == null) {
+            return Mono.error(new IllegalArgumentException("Parameter serviceParam is required and cannot be null."));
+        }
+        if (accessToken == null) {
+            return Mono.error(new IllegalArgumentException("Parameter accessToken is required and cannot be null."));
+        }
```

Comment:
Was this a code generation update? If not this will be overwritten the next time this is regenerated.

Reply:
This was actually me trying to play with some autogen parameters and see what some of the arguments are used for. will revert